app/uk/gov/hmrc/emcstfe/services/GetMovementService.scala

```diff
@@ -46,10 +46,12 @@ class GetMovementService @Inject()(
   def getMovement(getMovementRequest: GetMovementRequest, forceFetchNew: Boolean)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Either[ErrorResponse, GetMovementResponse]] = {
     repository.get(getMovementRequest.arc).flatMap {
       case Some(value) =>
-        logger.info("[getMovement] Matching movement found, calling GetMovementIfChanged")
+        logger.info("[getMovement] Matching movement found")
         if (forceFetchNew) {
+          logger.info("[getMovement] GetMovementIfChanged")
           getMovementIfChanged(getMovementRequest, value)
         } else {
+          logger.info("[getMovement] generateGetMovementResponse from cached movement")
```

Comment:
Just thinking, would we want these as `debug`? Won't this get quite chatty in Production?